### ISSUE TITLE
[CY-4186] Setting the base version

### ISSUE THREAD

**Issue by CWSites** [codacy/git-version#48]
Where does the `Previous Version` come from in the output? I've tried each of the following and it still uses `0.0.0` as the previous version. I would like to be able to specify the starting point because we already have version numbers in place and I want this action to take over the role of manually doing it.

* `./package.json`
* `./mono-repo/directory/package.json`
* github tags
* github releases

I'm fairly new to this but I feel like this would be a common need, so perhaps I'm missing something?

**Comment by boswelja** [comment]
I dug a little into the source and it looks like it pulls versions from the last tag (if possible)

**Comment by lolgab** [comment]
@CWSites You can tag the commit of the last release and then git-version will give you a successive tag correctly.

```bash
git checkout previous_commit_to_tag
git tag x.y.z
git push origin x.y.z
```

**Comment by CWSites** [comment]
@lolgab I was able to get the action to work and properly tag my repo with `0.0.1` and I used another command to add `-alpha` as a suffix to the github tag so that it is properly tagged `0.0.1-alpha` which is the format we are following. However the second time I made a change and ran the action the version didn't bump properly and remained `0.0.1`. Is this because I'm using the suffix `-alpha`? Is there a way to do a suffix instead of a prefix?
